pom.xml

```diff
@@ -81,7 +101,24 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>3.8.1</version>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
+
+
+
+    <!--  https://github.com/shamsoftware/sham-ssh  :Mock SSH Server used for testing-->
+    <dependency>
+      <groupId>software.sham</groupId>
+      <artifactId>sham-ssh</artifactId>
+      <version>0.1.0</version>
```

Comment:
Add a `<scope>test</scope>` for test only dependency

Reply:
<scope> added